kk/views/hearing.py

```diff
@@ -78,3 +79,8 @@ def follow(self, request, pk=None):
 
         # return success
         return response.Response({'status': 'You follow a hearing now'}, status=status.HTTP_201_CREATED)
+
+    @detail_route(methods=['get'])
+    def report(self, request, pk=None):
```

Comment:
This should probably have some sort of special authentication. I'll make an issue of that.